posts/templates/posts/create_post.html

```diff
@@ -0,0 +1,27 @@
+{% load static from staticfiles %}
```

Comment:
Use template inheritance for common markup. Make a parent template with blocks for title and content

Reply:
okay i will use template inheritance